resolwe_bio/management/commands/generate_de.py

```diff
@@ -0,0 +1,157 @@
+import gzip
+import json
+import os
+import random
+import shutil
+import string
+import datetime
+import json
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.utils import timezone
+from resolwe.flow.models import Data, Process, Storage
+from resolwe_bio.models import Sample
+from resolwe_bio.tools.utils import escape_mongokey, gzopen
+
+class Command(BaseCommand):
+
+    """Generate test differential expression data"""
+
+    help = "Generate differential expressions"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-n', '--n-diffexps', type=int, default=4,
+                            help="Number of differential expressions to generate (default: %(default)s)")
+        parser.add_argument('-g', '--group-size', type=int, default=5,
+                            help="Number of samples in case/control group per DE (default: %(default)s)")
+
+    @staticmethod
+    def get_random_word(length):
+        return ''.join(random.choice(string.lowercase) for _ in range(length))
+
+    @staticmethod
+    def generate_expressions(gene_ids, path):
+        """Generate random expression data"""
+        genes = {}
+        with gzip.open(os.path.join(path, 'expressions.tab.gz'), 'wb') as f:
+            f.write('{}\t{}\n'.format('Gene', 'Expression'))
+            with gzopen(gene_ids) as gene_ids:
+                all_genes = [line.strip() for line in gene_ids]
+                for gene in all_genes:
+                    expression = random.gammavariate(1, 100)
+                    f.write('{}\t{}\n'.format(gene, expression))
+                    genes[escape_mongokey(gene)] = expression
+
+        with open(os.path.join(path, 'expressions.json'), 'w') as json_file:
+            js_out = '{{"exp_json":{}}}'.format(json.dumps({'genes': genes}, separators=(',', ':')))
+            json_file.write(js_out)
+
+    def set_user(self):
```

Comment:
This is duplicated function from `generate_etc` file (Janez just renamed it to `get_superuser`). Move common functions to `utils.py` in `commands` directory.